css/css-text-decor/text-decoration-line-through-wavy-covers-whole-line-length-001.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Decoration Test: Wavy line-through covers the whole line length</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-text-decor-4/#text-decoration-line-property">
+<meta name="assert" content="Checks that a wavy line-through covers the whole line lenght.">
+<link rel="mismatch" href="reference/text-decoration-wavy-covers-whole-line-length-001-ref.html">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+<style>
+  body {
+    overflow: hidden;
+  }
+  #wrapper {
+    position: relative;
+    margin: 100px;
+  }
+  #decoration {
+    font: 400px/1 Ahem;
+    color: transparent;
+    text-decoration-line: line-through;
+    text-decoration-color: green;
+    text-decoration-style: wavy;
+    text-decoration-skip-ink: none;
+  }
+  #abspos {
+    position: absolute;
+    left: -100px;
+    top: -100px;
```

Comment:
can you reduce these top values a bit to avoid the white box touching the \<p> text? if the white box covers some antialiasing pixels along the bottom edge (correct me if that’s impossible), these tests will always pass.

Reply:
I've set `positon: absolute` on the `#wrapper` so there's no margin collapsing, and we shouldn't have this issue at all. Nice catch!